example/main.tf

```diff
@@ -2,14 +2,19 @@ module "cloudfront_auth" {
   source = "../"
 
   auth_vendor             = "cognito"
+  authz                   = "2"  # Is this a number or a name?
```

Comment:
As stated in original description at [variables.tf](https://github.com/scalefactory/terraform-cloudfront-auth/blob/283c2c9179dec049e2946fc3a58b7014b5289c44/variables.tf#L41)
this is applicable only for microsoft or google auth_vendor (defines additional params )